From f48d443c945a4384bfff8925620ad513e32f3bcb Mon Sep 17 00:00:00 2001
From: k2wl <k2wl@github.com>
Date: Sat, 26 Jul 2014 09:23:37 +0530
Subject: [PATCH] Add support for APM_SIM_POWER_DOWN Fix compilation issue due
 to REASON_APM_SIM_POWER_DOWN not found. make it compatible with PATCH " Fix
 SIM activation issue for single sim during APM toggle." in telephony-msim

Change-Id: I61e79c32ba4261cb81b74838244f1dc5dc8b977e
Signed-off-by: k2wl <k2wl@github.com>
---
 telephony/java/com/android/internal/telephony/MSimConstants.java | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/telephony/java/com/android/internal/telephony/MSimConstants.java b/telephony/java/com/android/internal/telephony/MSimConstants.java
index 9d35fd3..a878926 100644
--- a/telephony/java/com/android/internal/telephony/MSimConstants.java
+++ b/telephony/java/com/android/internal/telephony/MSimConstants.java
@@ -57,6 +57,7 @@ public class MSimConstants {
     public enum CardUnavailableReason {
         REASON_CARD_REMOVED,
         REASON_RADIO_UNAVAILABLE,
-        REASON_SIM_REFRESH_RESET
+        REASON_SIM_REFRESH_RESET,
+        REASON_APM_SIM_POWER_DOWN
     };
 }
-- 
1.9.1

